docs/release-notes/snapcraft-8-10.rst

```diff
@@ -0,0 +1,193 @@
+.. _release-8.10:
+
+Snapcraft 8.10 release notes
+============================
+
+7 July 2025
+
+Learn about the new features, changes, and fixes introduced in Snapcraft 8.10.
+
+
+Requirements and compatibility
+------------------------------
+
+See :ref:`reference-system-requirements` for information on the minimum hardware and
+installed software.
+
+
+What's new
+----------
+
+Snapcraft 8.10 brings the following features, integrations, and improvements.
+
+
+.NET plugin (v2)
+~~~~~~~~~~~~~~~~
+
+The :ref:`craft_parts_dotnet_v2_plugin` is now available for core24 snaps. We've
+introduced a new version of the plugin that is easier to use and configure.
+
+Older bases will continue to use the :ref:`previous version <craft_parts_dotnet_plugin>`
+of the dotnet plugin.
+
+Init profile for ROS 2 applications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A new init profile, ``ros2``, has been added. This profile makes it easier for developers
+to create a ROS 2 snap.
```

Comment:
```suggestion
to create an ROS 2 snap.
```
We may have discussed this before, but this is how we've used the article in the docs.

Reply:
https://github.com/canonical/snapcraft/pull/5575/commits/b74209193b1fba0ea13b8ccf46723cc5fd7a62fe